tests/conftest.py

```diff
@@ -35,3 +42,26 @@ async def scaffold_piccolo() -> AsyncGenerator:
     await create_db_tables(*TABLES)
     yield
     await drop_db_tables(*TABLES)
+
+
+#############################
+# Session Middleware Fixtures
+#############################
+async def mock_asgi_app(scope: "Scope", receive: "Receive", send: "Send") -> None:
+    pass
+
+
+@pytest.fixture()
+def session_middleware() -> SessionMiddleware:
```

Comment:
My thinking was not only to expose these as internal test fixtures, but also expose to the user a way to easily create sessions in tests.

Reply:
Something like whenever the session middleware is used in tests, it must implicitly starts with a default session?

Reply:
dunno, if you have a solution go for it.

Reply:
The best way is to have test `scope`, test `receive` and test `send` context managers inside `starlite.testing`. They will accept user arguments and default arguments to initialize them with user data. We will set a default value for `scope['session']` as well.

Reply:
Sounds good @infohash

Reply:
Merge this PR, I will raise a separate PR for this.